test/test_harness_aten_cuda.h

```diff
@@ -23,16 +23,16 @@
 #include <gflags/gflags.h>
 #include <gtest/gtest.h>
 
-#include <ATen/ATen.h>
+#include "tc/aten/aten.h"
 
 #include <cuda_runtime_api.h>
 
-#include "tc/aten/aten_compiler.h"
+#include "tc/aten/aten_compiler_new_api.h"
 #include "tc/core/cuda/cuda.h"
-#include "tc/core/cuda/cuda_tc_executor.h"
+#include "tc/core/cuda/cuda_tc_executor_new_api.h"
 #include "tc/core/flags.h"
 
-#include "test_harness_aten.h"
+#include "../../test/test_harness_aten.h"
```

Comment:
eh, why?  doing `../../` in the `./test` will get you outside of tc root dir.  These two files are next to each other in the hierarchy...

Reply:
wrong rebase, thanks!